OrbitGl/App.cpp

```diff
@@ -873,6 +873,7 @@ void OrbitApp::OnProcessSelected(int32_t pid) {
   std::shared_ptr<Process> process = FindProcessByPid(pid);
```

Comment:
ok, one small thing about commit message

Move link to the bug into Bug: stanza

At the end of commit message add
Bug: b/42
Test: <describe how you tested it>

Reply:
Okay done.